Workbooks/CosmosDb/Resource Insights/Resource Insights.workbook

```diff
@@ -179,17 +227,31 @@
               ],
               "timeContextFromParameter": "TimeRange",
               "timeContext": {
-                "durationMs": 0
+                "durationMs": 86400000
               },
               "metrics": [
                 {
                   "namespace": "microsoft.documentdb/databaseaccounts",
                   "metric": "microsoft.documentdb/databaseaccounts-Requests-TotalRequests",
                   "aggregation": 7,
-                  "splitBy": "StatusCode"
+                  "splitBy": null
                 }
               ],
               "title": "Total Requests",
+              "filters": [
```

Comment:
![image](https://user-images.githubusercontent.com/7664931/110860814-1b401800-8272-11eb-9984-2433c1e86041.png)
Can also play around with the idea of adding the DB/Container to the title? What do you think about that? If you want to format it to readable text on the title, you can also look at using criteria params and you can set rules lilke, if DB is empty, then the value is an empty string. If DB has a value, then the value is (Database: {Database})